bytecode/concrete.py

```diff
@@ -279,15 +278,19 @@ def _assemble_lnotab(first_lineno, linenos):
 
         return b''.join(lnotab)
 
+    def compute_stacksize(self):
+        bytecode = self.to_bytecode()
+        cfg = _bytecode.ControlFlowGraph.from_bytecode(bytecode)
+        return cfg.compute_stacksize()
+
     def to_code(self):
         code_str, linenos = self._assemble_code()
         lnotab = self._assemble_lnotab(self.first_lineno, linenos)
         nlocals = len(self.varnames)
```

Comment:
Please call compute_stacksize() and store result in a variable. It can help if compute_stacksize() raises an exception?